zio-kafka-test-utils/src/main/scala/zio/kafka/embedded/Kafka.scala

```diff
@@ -55,5 +60,34 @@ object Kafka {
     ZIO.acquireRelease(ZIO.attempt(Kafka.Sasl(EmbeddedKafkaService(EmbeddedKafka.start()))))(_.value.stop())
   }
 
+  val sslEmbedded: ZLayer[Any, Throwable, Kafka] = ZLayer.scoped {
+    val listener = s"${SecurityProtocol.SSL}://localhost:$defaultKafkaPort"
+
+    val keyStorePath   = Paths.get(Kafka.getClass.getResource("/keystore/kafka.keystore.jks").toURI).toFile
+    val trustStorePath = Paths.get(Kafka.getClass.getResource("/truststore/kafka.truststore.jks").toURI).toFile

```

Comment:
Since these 2 things are repeated here and in  `KafkaTestUtils`, I'd put `private[zio.kafka]` functions in `KafkaTestUtils` and use them here, WDYT?

Reply:
why not)

Reply:
fixed